test/LeanCode.IntegrationTests/PushNotificationTokenStoreTests.cs

```diff
@@ -18,7 +18,7 @@ public PushNotificationTokenStoreTests()
         app = new TestApp();
     }
 
-    [Fact]
+    [IntegrationFact]
```

Comment:
I guess we need to adjust docs on how to run those tests.

Reply:
True, will do.